src/gui/src/settings/options-window.cpp

```diff
@@ -283,6 +287,16 @@ OptionsWindow::OptionsWindow(Profile *profile, ThemeLoader *themeLoader, QWidget
 	ui->comboImagePositionVideoH->setCurrentIndex(positionsH.indexOf(settings->value("imagePositionVideoH", "left").toString()));
 	ui->lineImageBackgroundColor->setText(settings->value("imageBackgroundColor", QString()).toString());
 
+	ui->comboActionButtons->clear();
```

Comment:
Rather than clearing everything and re-adding values, you could just use `setItemData()`.